src/components/Cart.js

```diff
@@ -90,26 +116,42 @@ function Cart(props) {
           updateFlowerInCart={updateFlowerInCart}
           removeFlowerFromCart={removeFlowerFromCart}
         />
+        <hr></hr>
       </div>
     );
   });
   return (
     <div className="bg-gray-100 h-screen pt-12">
       <div className="m-auto w-11/12 flex flex-col">
         {/* header */}
-        <div className="h-28 flex items-center justify-between bg-white">
+        <div className="h-28 mb-10 flex items-center justify-center bg-white">
           <p className="pl-8 text-4xl text-gray-700 font-semibold p-4">
             Shopping Cart
           </p>
-          <img
-            className="pt-6 pr-20 h-16"
-            src="/assets/icons/CartGray.svg"
-            alt="cart"
-          />
         </div>
-
         {/* cards */}
-        {items}
+        <div className="flex justify-between ">
+          <div>{items}</div>
+          <div className="flex flex-col justify-around items-center w-1/3 h-60 text-center text-xl bg-white">
+            <img
+              className="pt-6 h-16"
+              // src="/assets/icons/CartGray.svg"
+              src="/assets/icons/Cart64x40.svg"
+              alt="cart"
+            />
+            <p>
+              Subtotal ({qtySum} items){" "}
+              <span className="font-semibold">${subTotal}</span>
+            </p>
```

Comment:
I think the count of items listed should be the number of line items (flower types) being ordered, not the total quantity across all line items.

Reply:
![image](https://user-images.githubusercontent.com/58010612/104760846-98925500-5727-11eb-8f96-fa3e502b6e0c.png)

Reply:
I was following See's candies and amazon's cart page, they use total number of items. I think we should follow that design.

Reply:
I see now I misunderstood the usage of this field and I agree it should match those other sites.